### PR TITLE
Changed severity in logs from id to label

### PR DESCRIPTION
**Pull request by KateMengjunXia**
Added a flag "javaJSONOutput" to refinements.js, and set it in LogMessage.js, so that the "severity" in logs are labels instead of id's.

### CODE REVIEW

File: src/foam/nanos/logger/LogMessage.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ Implement LastModifiedByAware to suppress 'modified by' comment in journal outpu
       name: 'severity',
       class: 'Enum',
       of: 'foam.nanos.logger.LogLevel',
-      toJSON: function(value) { return value && value.label; }
+      toJSON: function(value) { return value && value.label; },
+      javaJSONOutput: `((foam.nanos.logger.LogLevel) value).getLabel()`
```
Comment (adamvy) [inline src/foam/nanos/logger/LogMessage.js]: Will this break parsing of the journal entries back into the system?  I don't believe we parse labels into enums in java currently.

What's the motivation for this change?

Comment (kgrgreer) [reply src/foam/nanos/logger/LogMessage.js]: Implementing fromJSON as well would fix parsing.

Comment (jlhughes) [reply src/foam/nanos/logger/LogMessage.js]: The motivation in this particular case to have have human readable log messages.  So severity shows as 'debug', 'error' rather than 0, or 5 for example. 
In this case we are explicitly using a WriteOnlyJDAO with no intent, in the short term, for reading it back.  But yes, should we need/want to fromJSON would be required.   Makes sense to add it now so it's complete if we later want an external log viewer, which would be the case if log messages are all sent to a NOC. 
@KateMengjunXia please implement fromJSON.

**Comment by KateMengjunXia** [comment]
I generated the following code in LogMessage.java, let me know what you think:
```
      public foam.nanos.logger.LogLevel forLabel(String label) {
        return foam.nanos.logger.LogLevel.forLabel(label);
      }
```

```
      public Object fromJSON(String value) {
        return forLabel(value);
      }
```

**Comment by jlhughes** [comment]
I would just simplify it with:
```
      public Object fromJSON(String value) {
        return foam.nanos.logger.LogLevel.forLabel(label);
      }

```
If in the future there are other calls to forLabel you could pull it out then.

**Review by jlhughes — APPROVED** [review]
